lib/rfc5424.js

```diff
@@ -35,7 +35,7 @@ module.exports = function ($options) {
     const hostname = data.hostname
     const appname = (options.appname !== 'none') ? options.appname : '-'
     const msgid = (data.req && data.req.id) ? data.req.id : '-'
-    const structuredData = '-'
+    const structuredData = (options.structuredData !== 'none') ? options.structuredData : '-'
```

Comment:
Why check for a string`'none'` instead of a falsy value here? e.g:

```js
const structuredData = options.structuredData  || '-'
```

Reply:
That's how I'd write it normally but instead copied the style you used with `appname` as I figured you had a reason for defining configuration options that way. Now that you mention it I'm curious why you did it like that?

Reply:
Oops, it looks like I did do that. It's been so long since I wrote this. I'm fine with you correcting the style.